code/modules/projectiles/projectile/beams.dm

```diff
@@ -52,8 +52,12 @@ var/list/beam_master = list()
 		var/broken
 		var/atom/curr = current
 		var/Angle=round(Get_Angle(firer,curr))
-		var/icon/I=new('icons/obj/zap.dmi',"lightning")
-		I.Turn(Angle)
+		var/icon/I=new('icons/obj/lightning.dmi',initial(icon_state))

```

Comment:
One other thing that would be nice s removing the initial here, in this case there's pretty much no reason to use it over just putting the actual string here